From c6a12704979102c3e79c0283ededb12e5538b178 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:41 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(a63c7ce)..lineage-16.0-android-9.0.0_r31(edcf0d5) (DO NOT
 SUBMIT)

Bill Yi (4):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (3):
 - Snap for 5073521 from a7e25bbef0b73500b275da2dcbc9ebcfee4b4981 to
   pi-qpr2-release
 - Snap for 5115288 from 33817b4fd02f0a8b585923a75f26c2a1fb19b4ad to
   pi-qpr2-release
 - Snap for 5130385 from c193436a5e829771b313c48b0c3980af56c394e9 to
   pi-qpr2-release

Change-Id: Ia1175bd12b653ac220b32da150efc7bbc671b464

Change-Id: I2b19c29fb3a5782399fa9c0b9a3a4cc7fb956e56
---
 res/values-ar/strings.xml | 14 +++++++-------
 res/values-ca/strings.xml |  2 +-
 res/values-ne/strings.xml |  4 ++--
 res/values-ta/strings.xml |  2 +-
 4 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 1fc575f..f252f33 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -16,14 +16,14 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="963247385700831716">"مدير سعة التخزين"</string>
+    <string name="app_name" msgid="963247385700831716">"مدير مساحة التخزين"</string>
     <string name="cancel" msgid="7021218262867558825">"إلغاء"</string>
     <string name="storage_menu_free" msgid="1878247401436882778">"إخلاء بعض المساحة"</string>
     <string name="deletion_helper_title" msgid="3526170325226275927">"إزالة العناصر"</string>
     <string name="deletion_helper_app_summary" msgid="4904590040180275237">"قبل <xliff:g id="DAYS">%1$d</xliff:g> يوم"</string>
     <!-- no translation found for deletion_helper_app_summary_item_size (3770886184921427886) -->
     <skip />
-    <string name="deletion_helper_app_summary_never_used" msgid="2695210890063792461">"لم يتم استخدام سعة التخزين في العام الماضي"</string>
+    <string name="deletion_helper_app_summary_never_used" msgid="2695210890063792461">"لم يتم استخدام مساحة التخزين في العام الماضي"</string>
     <string name="deletion_helper_app_summary_unknown_used" msgid="8576377054665785558">"وقت آخر استخدام لسعة التخزين غير مؤكَّد"</string>
     <string name="deletion_helper_free_button" msgid="1760529213407548661">"إخلاء <xliff:g id="FREEABLE">%1$s</xliff:g>"</string>
     <string name="deletion_helper_photos_title" msgid="2602723121486729972">"الصور والفيديوهات التي تم الاحتفاظ بنسخة احتياطية منها"</string>
@@ -38,8 +38,8 @@
     <string name="deletion_helper_clear_dialog_title" msgid="9169670752655850967">"إخلاء بعض المساحة"</string>
     <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"ستتم إزالة <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> من المحتوى من جهازك."</string>
     <string name="deletion_helper_clear_dialog_remove" msgid="863575755467985516">"إخلاء بعض المساحة"</string>
-    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"هل تريد إدارة سعة التخزين تلقائيًا؟"</string>
-    <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"أصبح خاليًا الآن <xliff:g id="USED">%1$s</xliff:g>. هل تسمح لمدير سعة التخزين بالتفريغ التلقائي للمساحة بإزالة المحتوى من جهازك بعد أن احتفظت بنسخة احتياطية منه؟"</string>
+    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"هل تريد إدارة مساحة التخزين تلقائيًا؟"</string>
+    <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"أصبح خاليًا الآن <xliff:g id="USED">%1$s</xliff:g>. هل تسمح لمدير مساحة التخزين بالتفريغ التلقائي للمساحة بإزالة المحتوى من جهازك بعد أن احتفظت بنسخة احتياطية منه؟"</string>
     <string name="deletion_helper_upsell_cancel" msgid="7084167642850053889">"لا، شكرًا"</string>
     <string name="deletion_helper_upsell_activate" msgid="7157360257760895976">"تشغيل"</string>
     <string name="deletion_helper_apps_group_title" msgid="2578309121350341258">"التطبيقات المستخدمة قليلاً"</string>
@@ -48,10 +48,10 @@
     <string name="deletion_helper_automatic_title" msgid="4610119358490679299">"تلقائي"</string>
     <string name="deletion_helper_manual_title" msgid="3905192395039688529">"يدويًا"</string>
     <string name="deletion_helper_preference_title" msgid="803484286872745892">"المساحة الخالية الآن"</string>
-    <string name="automatic_storage_manager_service_label" msgid="648592272375278978">"خدمة إدارة سعة التخزين تلقائيًا"</string>
+    <string name="automatic_storage_manager_service_label" msgid="648592272375278978">"خدمة إدارة مساحة التخزين تلقائيًا"</string>
     <string name="automatic_storage_manager_notification_title" product="default" msgid="8480483265300964701">"مساحة الفارغة في هاتفك منخفضة"</string>
     <string name="automatic_storage_manager_notification_title" product="tablet" msgid="3150907918531844467">"أوشكت المساحة على النفاد في جهازك اللوحي"</string>
-    <string name="automatic_storage_manager_notification_summary" msgid="2267399886362245822">"السماح لمدير سعة التخزين بالتفريغ التلقائي للمساحة بإزالة الصور والفيديوهات القديمة عندما توشك مساحة التخزين في هاتفك على الامتلاء مرة أخرى."</string>
+    <string name="automatic_storage_manager_notification_summary" msgid="2267399886362245822">"السماح لمدير مساحة التخزين بالتفريغ التلقائي للمساحة بإزالة الصور والفيديوهات القديمة عندما توشك مساحة التخزين في هاتفك على الامتلاء مرة أخرى."</string>
     <string name="automatic_storage_manager_cancel_button" msgid="1826365037494962803">"لا، شكرًا"</string>
     <string name="automatic_storage_manager_activate_button" msgid="6257070263061898041">"تشغيل"</string>
     <string name="deletion_helper_photos_loading_title" msgid="2768067991066779772">"الصور والفيديوهات التي تم الاحتفاظ بنسخة احتياطية منها"</string>
@@ -59,7 +59,7 @@
     <string name="deletion_helper_no_threshold" msgid="6943179204098250444">"عرض كل العناصر"</string>
     <string name="deletion_helper_default_threshold" msgid="8410389370069021113">"إخفاء العناصر المستخدَمة حديثًا"</string>
     <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"ستتم إزالة <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> من المحتوى من جهازك."</string>
-    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"تتم إدارة سعة التخزين لجهازك الآن عن طريق مدير سعة التخزين"</string>
+    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"تتم إدارة مساحة التخزين لجهازك الآن عن طريق مدير مساحة التخزين"</string>
     <string name="empty_state_title" msgid="4033285438176545309">"ما من عناصر للإزالة"</string>
     <string name="empty_state_review_items_link" msgid="8411186441239304545">"مراجعة العناصر المستخدَمة حديثًا"</string>
     <string name="empty_state_summary" msgid="8439893007424243790">"ليست هناك أي ملفات قديمة لإزالتها. يمكنك إزالة الصور والفيديوهات والتطبيقات المستخدمة حديثًا من أجل إخلاء مساحة."</string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index bd2f5c2..ff03c09 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -42,7 +42,7 @@
     <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"Espai d\'emmagatzematge lliure: <xliff:g id="USED">%1$s</xliff:g>. Vols permetre que el gestor d\'emmagatzematge alliberi espai automàticament suprimint del dispositiu les còpies de seguretat del contingut?"</string>
     <string name="deletion_helper_upsell_cancel" msgid="7084167642850053889">"No, gràcies"</string>
     <string name="deletion_helper_upsell_activate" msgid="7157360257760895976">"Activa"</string>
-    <string name="deletion_helper_apps_group_title" msgid="2578309121350341258">"Aplicacions poc utilitzades"</string>
+    <string name="deletion_helper_apps_group_title" msgid="2578309121350341258">"Aplicacions utilitzades amb poca freqüència"</string>
     <!-- no translation found for deletion_helper_apps_group_summary (360315517252452356) -->
     <skip />
     <string name="deletion_helper_automatic_title" msgid="4610119358490679299">"Automàtica"</string>
diff --git a/res/values-ne/strings.xml b/res/values-ne/strings.xml
index 26d525d..ec10b66 100644
--- a/res/values-ne/strings.xml
+++ b/res/values-ne/strings.xml
@@ -36,7 +36,7 @@
     <!-- no translation found for deletion_helper_downloads_summary_empty (3988054032360371887) -->
     <skip />
     <string name="deletion_helper_clear_dialog_title" msgid="9169670752655850967">"ठाउँ खाली गर्नुहोस्"</string>
-    <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"सामग्रीको <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> लाई तपाईंको यन्त्रबाट हटाइनेछ"</string>
+    <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"सामग्रीको <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> लाई तपाईंको यन्त्रबाट हटाइने छ"</string>
     <string name="deletion_helper_clear_dialog_remove" msgid="863575755467985516">"ठाउँ खाली गर्नुहोस्"</string>
     <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"भण्डारणको स्वतः व्यवस्थापन गर्ने हो?"</string>
     <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"अहिले <xliff:g id="USED">%1$s</xliff:g> खाली छ। भण्डारण प्रबन्धकलाई तपाईँको यन्त्रबाट ब्याक अप गरिएको सामग्री हटाएर स्वतः ठाउँ खाली गर्न दिने हो?"</string>
@@ -58,7 +58,7 @@
     <string name="deletion_helper_photos_loading_summary" msgid="8203033249458245854">"वस्तुहरू भेट्टाउँदै…"</string>
     <string name="deletion_helper_no_threshold" msgid="6943179204098250444">"सबै वस्तुहरू देखाउनुहोस्"</string>
     <string name="deletion_helper_default_threshold" msgid="8410389370069021113">"हालैका वस्तुहरू लुकाउनुहोस्"</string>
-    <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"सामग्रीको <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> लाई तपाईंको यन्त्रबाट हटाइनेछ"</string>
+    <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"सामग्रीको <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> लाई तपाईंको यन्त्रबाट हटाइने छ"</string>
     <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"तपाईंको भण्डारण अहिले भण्डारण प्रबन्धकद्वारा व्यवस्थापन भइरहेको छ"</string>
     <string name="empty_state_title" msgid="4033285438176545309">"हटाउनु पर्ने कुनै कुरा छैन"</string>
     <string name="empty_state_review_items_link" msgid="8411186441239304545">"हालैका वस्तुहरूको समीक्षा गर्नुहोस्"</string>
diff --git a/res/values-ta/strings.xml b/res/values-ta/strings.xml
index 979a12d..4c0d473 100644
--- a/res/values-ta/strings.xml
+++ b/res/values-ta/strings.xml
@@ -38,7 +38,7 @@
     <string name="deletion_helper_clear_dialog_title" msgid="9169670752655850967">"இடத்தைக் காலியாக்கு"</string>
     <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"உங்கள் சாதனத்திலிருந்து <xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g> அளவிலான உள்ளடக்கம் அகற்றப்படும்"</string>
     <string name="deletion_helper_clear_dialog_remove" msgid="863575755467985516">"இடத்தைக் காலியாக்கு"</string>
-    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"சேமிப்பகத்தைத் தானாகவே நிர்வகிக்கவா?"</string>
+    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"சேமிப்பிடத்தைத் தானாகவே நிர்வகிக்கவா?"</string>
     <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"இப்போது <xliff:g id="USED">%1$s</xliff:g> இடம் காலியாக உள்ளது. காப்புப் பிரதி எடுத்த உள்ளடக்கத்தை உங்கள் சாதனத்திலிருந்து அகற்றி இடத்தைக் காலியாக்க, சேமிப்பக நிர்வாகியை அனுமதிக்கவா?"</string>
     <string name="deletion_helper_upsell_cancel" msgid="7084167642850053889">"வேண்டாம்"</string>
     <string name="deletion_helper_upsell_activate" msgid="7157360257760895976">"இயக்கு"</string>
-- 
2.17.1

